src/main/java/com/powsybl/openloadflow/network/LfBus.java

```diff
@@ -118,7 +117,7 @@ default double getHighVoltageLimit() {
 
     Optional<LfShunt> getControllerShunt();
 
-    LfLoads getLfLoads();
+    LfLoads getLoads();
```

Comment:
You really prefer that name? It is a bit confusing with the `Load` of IIDM network, no?

Reply:
For any other element accessors we did not use lf